packages/components/src/index.ts

```diff
@@ -51,8 +51,6 @@ export {
   makeCSSVariableTheme,
   mapLeafsOfObject,
 } from "@kaizen/design-tokens"
-export * from "@kaizen/design-tokens/src/themes"
-export * from "@kaizen/design-tokens/src/types"
```

Comment:
Was this referencing `themes/types.ts`? Or are we missing something?

Reply:
Oh you're right, that doesn't exist haha, it should've been `types.ts` yes. Another way to tackle the issue might've been to changes these to export from `dist/themes` and not `src`. But I chose the nuclear option lol

Reply:
Perhaps exporting these from `dist` may have prevented the problem? 😂

Reply:
We'll never know now 😈